packages/plugins/rubygem-foreman_kubevirt/rubygem-foreman_kubevirt.spec

```diff
@@ -0,0 +1,108 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
```

Comment:
this is missing the

```
# Generated from <%= package.spec.file_name %> by gem2rpm -*- rpm-spec -*-
# template: foreman_plugin
```

header, which might confuse our toolinglater on

Reply:
It's mostly the `# template: foreman_plugin` one that matters since our tooling does use that. The `Generated from ...` is actually old and IMHO by now out of date. We add a changelog entry that includes this info as well.

Reply:
Added both, please update if you'd like the first line be removed in this PR or if you plan to do it cross-repo at some point for all spec files.

Reply:
As long as the `template` line is there, I don't care too much for the other one, so let's leave it as is for now.